server/controllers/addUser.js

```diff
@@ -2,20 +2,25 @@ const { hashPassword, signupSchema } = require('../utils/validations');
 const { addUserQuery, getUserId } = require('../database/queries');
 
 const addUser = (req, res, next) => {
-  const { error, value: { username, email, password } } = signupSchema.validate(req.body);
+  const {
+    error, value: {
+      username, email, password, userRole,
+    },
+  } = signupSchema.validate(req.body);
```

Comment:
try to validate your body asynchronously, also if you want validation to continue even If your first validator failed, add `{abortEarly: false}` as options to your validate